outport/process/alteredaccounts/alteredAccountsProvider.go

```diff
@@ -211,6 +209,22 @@ func (aap *alteredAccountsProvider) addTokensDataForMarkedAccount(
 	return nil
 }
 
+func (aap *alteredAccountsProvider) convertMetaData(metaData *esdt.MetaData) *outportcore.TokenMetaData {
```

Comment:
add unit test for this function

Reply:
Extended test `testExtractAlteredAccountsFromPoolShouldIncludeDestinationFromTokensLogsTop` to check for all filled fields in token meta data